client/src/components/HostViews/Tutorial.vue

```diff
@@ -65,10 +72,65 @@ export default {
         setTimeout(() => this.next(), 1500)
       }
     },
+    addVotesToDisplay() {
+      let totalVotes = this.votesForSkip + this.votesAgainstSkip
+      let percentAgainst = Math.round((this.votesAgainstSkip / totalVotes) * 100)
+      let diff = percentAgainst - this.againstPercentOnDisplay
+      if (!diff) return
```

Comment:
Are you doing bool comparisons on an integer?

Reply:
0 has is a falsely value in JS. If the diff is 0 we return as no adjustment needs to be made in the display of votes